### PR TITLE
Adds a Github Service definition

### PR DESCRIPTION
**Pull request by smaye81**
This adds a protobuf definition for a Github Service to interact with Github integration information.

### CODE REVIEW

File: proto/buf/alpha/registry/v1alpha1/github.proto [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+message GithubAppConfig {
+    string app_id = 1;
+}
+
+message GetGithubAppConfigRequest{
+    string id = 1;
```
Comment (smaye81) [inline proto/buf/alpha/registry/v1alpha1/github.proto:24]: I'm guessing `id` is not a good name, but not sure what this ID represents.

**Comment by smaye81** [comment]
@SyrieBianco @srikrsna-buf Adding you two for now just to make sure I'm on the right track.  I'm sure there's some conventions I'm not following, so wanted to let you see it before I went to far in code.

File: proto/buf/alpha/registry/v1alpha1/github.proto (hunk 1 above)
Comment (srikrsna-buf) [reply proto/buf/alpha/registry/v1alpha1/github.proto:24]: For now we can leave it blank as it should only have one entry per deployment.

**Comment by bufdev** [comment]
Should this all be in the private APIs, or the public ones?

**Comment by srikrsna-buf** [comment]
It is for the admin panel integration

File: proto/buf/alpha/registry/v1alpha1/github.proto (hunk 1 above)
Comment (smaye81) [reply proto/buf/alpha/registry/v1alpha1/github.proto:24]: OK, meaning just an empty request object?

**Review by srikrsna-buf — APPROVED** [review]
Checked with @robbertvanginkel. These APIs go here. Looks good to me.

**Pull request merged**